core/app/models/spree/app_configuration.rb

```diff
@@ -360,8 +360,10 @@ def add_payment_sources_to_wallet_class
     # @return [Class] a class with the same public interfaces as
     #   Spree::Tax::OrderAdjuster
     # @api experimental
+    # @deprecated - use tax_calculator_class instead
     attr_writer :tax_adjuster_class
     def tax_adjuster_class
+      Spree::Deprecation.warn("Spree::AppConfiguration#tax_adjuster_class is DEPRECATED, please use #tax_calculator_class instead.", caller)
```

Comment:
If we keep this interface and move the new stuff into the default order adjuster? As per @jhawthorns comment on a later commit?